### ISSUE TITLE
Drift modular code generation  - MAPPED BY Could not load helpers

### ISSUE THREAD

**Issue by North101** [simolus3/moor#2183]
With modular code generation enabled:
```yaml
      drift_dev:
        enabled: false
      drift_dev:analyzer:
        enabled: true
        options: &options
          apply_converters_on_variables: true
          generate_values_in_copy_with: true
          named_parameters: true
          override_hash_and_equals_in_result_sets: true
          generate_connect_constructor: true
          store_date_time_values_as_text: true
          sql:
            dialect: sqlite
            options:
              known_functions:
                LOCALIZE: "TEXT (TEXT, TEXT NULL, TEXT NULL)"
      drift_dev:modular:
        enabled: true
        options: *options
```

Im getting the following error:
```
line 10, column 13: Could not load helpers
   ╷
10 │   name TEXT MAPPED BY `const LocalizedValueConverter()` NOT NULL,
```

**Comment by North101** [comment]
Looking a bit further, it seems the `.drift_prep.json` file isn't being created:
```
AssetNotFoundException: botc_helper|lib/db/sql/character_option.drift_prep.json
```

**Comment by simolus3** [comment]
Thanks for the report! Should be fixed in c0d5188c18ba101dca6079c1b86e08915041a10d.
